### ISSUE TITLE
what's y_cls & y_cls[i,T]?

### ISSUE THREAD

**Issue by MuseCP** [Lextal/pspnet-pytorch#9]
can someone show me the DataLoader? I don't know how to creat y_cls

**Comment by zhuruzhao** [comment]
@MuseCP have you worked it out? I have no idea how to create y_cls too

**Comment by MuseCP** [comment]
yes you need write a dataloeader yourself. return values are input，groundtruth，and a 1xn tensor contains class imformation。n is class num，1 is the label has the class and 0 is not




| |
翟墨
邮箱：zhm216@163.com
|

签名由 网易邮箱大师 定制

On 04/01/2019 19:27, 1535315854 wrote:

@MuseCP have you worked it out? I have no idea how to create y_cls too

**Comment by zhuruzhao** [comment]
@MuseCP Thanks, I have written the dataloader yesterday. I also changed a few lines to make the code run. The problem I am facing now is how to evaluate the results? Do I have to write the evaluate code by myself too?

**Comment by MuseCP** [comment]
yes I write the code myself too




| |
翟墨
邮箱：zhm216@163.com
|

签名由 网易邮箱大师 定制

On 04/02/2019 10:14, 1535315854 wrote:

@MuseCP Thanks, I have written the dataloader yesterday. I also changed a few lines to make the code run. The problem I am facing now is how to evaluate the results? Do I have to write the evaluate code by myself too?

**Comment by zhuruzhao** [comment]
@MuseCP So what's your model performance? Could you please share your evaluate code? Thanks a lot!

**Comment by MuseCP** [comment]
i did not write the evaluate part，only output the final result




| |
翟墨
邮箱：zhm216@163.com
|

签名由 网易邮箱大师 定制

On 04/02/2019 10:25, 1535315854 wrote:

@MuseCP So what's your model performance? Could you please share your evaluate code? Thanks a lot!
